src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -1011,6 +1011,10 @@ private void UpdateWMStyles(Action change)
                 style ^= (WindowStyles.WS_SIZEFRAME);
             }
 
+            MARGINS margins = new MARGINS();
+            margins.cyBottomHeight = _decorated == SystemDecorations.BorderOnly ? 1 : 0;
+            UnmanagedMethods.DwmExtendFrameIntoClientArea(_hwnd, ref margins);
```

Comment:
What happens on Win7 with a classic theme (aka DWM disabled)?

Reply:
Does Avalonia still (want to) support Win7 now that it has been officially retired?

Reply:
It still have hundreds of millions of users. By a huge margin more users than Desktop Linux and OSX. Those aren't going anywhere anytime soon. So, no, we aren't planning to drop Win7 support in the immediate future.t

Reply:
I don't have a windows 7 machine to test on, but dwm is only used to add the border for SystemDecorations.BorderOnly, so without dwm it should behave like SystemDecorations.None.